CHANGELOG.md

```diff
@@ -20,7 +21,7 @@ Desktop App v4.3.0 contains multiple medium level security fixes. [Upgrading](ht
 ### Improvements
 
 #### All Platforms
- - Added support for maintaining a user's online status while the desktop app is in the background but the user is interacting with their computer.
+ - Added support for maintaining a user's online status while the desktop app is in the background but the user is interacting with their computer. Requires Mattermost Server v5.16.0 or later.
```

Comment:
```suggestion
 - Added support for maintaining a user's online status while the desktop app is in the background but the user is interacting with their computer. Requires Mattermost Server v5.16.0, v5.15.1, v5.14.4, v5.9.5 or later.
```

Reply:
@deanwhillier is the above accurate? I don't think the v5.9 release supports it?

Reply:
That's correct, v5.9 doesn't support it as the feature was initially added to 5.14 so I didn't back port all the way to 5.9. The rest is correct.

Reply:
```suggestion
 - Added support for maintaining a user's online status while the desktop app is in the background but the user is interacting with their computer. Requires Mattermost Server v5.16.0, v5.15.1, v5.14.4 or later.
```